src/manage_sql_tls_certificates.c

```diff
@@ -826,11 +826,7 @@ delete_tls_certificates_user (user_t user)
 {
   /* Regular tls_certificate. */
 
-  sql ("DELETE FROM tls_certificate WHERE owner = %llu;", user);
-
-  /* Trash tls_certificate. */
-
-  sql ("DELETE FROM tls_certificate_trash WHERE owner = %llu;", user);
```

Comment:
Did you remove deleting the tls_certifcate_trash table by intention?

Reply:
Yes.  Timo removed the trashcan for TLS certs (to match the other secinfo).  I guess he just missed this statement.

Reply:
Ah thanks for the info!